spec/factories.rb

```diff
@@ -1,8 +0,0 @@
-# frozen_string_literal: true
-
-FactoryBot.define do
```

Comment:
I understand why you opted to delete this. And it bears pointing out that this sets you on a path of splitting different factories by purpose/model. Where will the `recipe` factory live?

(Useful piece of trivia: FactoryBot used to be called FactoryGirl and sometimes resources are under the old name).